src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -10,6 +10,44 @@ const itemsPropType = PropTypes.arrayOf(
   })
 );
 
+const renderLink = ({ cssClasses, createURL, refine, templateProps }) => (
+  item,
+  idx,
+  items
+) => {
+  const isLast = idx === items.length - 1;
+  const link = isLast ? (
+    item.name
+  ) : (
+    <a
+      className={cssClasses.link}
+      href={createURL(item.value)}
+      onClick={e => {
+        e.preventDefault();
+        refine(item.value);
+      }}
+    >
+      {item.name}
+    </a>
+  );
+
+  const itemClassnames = isLast
```

Comment:
You can use an object with `classNames` to do this kind of optional application of css classes:

```js
cx(cssClasses.item, {
  [cssClasses.selectedItem]: isLast,
})
```